### PR TITLE
virtio_port_hotplug: split host_proc_pid to fix the error

### PR DESCRIPTION
**Pull request by leidwang**
Signed-off-by: Leidong Wang <leidwang@redhat.com>

ID:1829188

### CODE REVIEW

**Comment by leidwang** [comment]
Hi @6-dehan, Could you help review this patch?Thanks a lot.
By the way, I have a question to confirm with you, we need to kill all processes in `host_proc_pid`? Or just kill the first process?

**Comment by 6-dehan** [comment]
@leidwang, LGTM, and run a loop with your patch, there is no further error. thanks for your effort.

**Comment by leidwang** [comment]
@YongxueHong Could you help review this patch?Thanks a lot.

**Comment by YongxueHong** [comment]
@leidwang 
Could you explain the reason to fix this error in the commit message?
Thanks,

**Comment by leidwang** [comment]
> @leidwang
> Could you explain the reason to fix this error in the commit message?
> Thanks,

Okay, the commit message has been updated.

File: qemu/tests/virtio_port_hotplug.py [hunk 1]
```diff
@@ -89,12 +89,15 @@ def run(test, params, env):
         session.close()
         host_script = params['host_script']
         check_pid_cmd = 'pgrep -f %s'
-        host_proc_pid = process.getoutput(check_pid_cmd % host_script, shell=True)
+        host_proc_pid = process.getoutput(check_pid_cmd % host_script,
+                                          shell=True)
         if host_proc_pid:
             logging.info("Kill the first serial process on host")
```
[the remark this answers is not in the document]
Comment (leidwang) [reply qemu/tests/virtio_port_hotplug.py:95]: Okay,I also have doubts about this place. Last time I discussed this issue with @6-dehan. We suspected that it was the PID left over from the previous case, so We decided to kill them all.
@6-dehan @nanliu-r Could you explain some details about this case?

[the remark this answers is not in the document]
Comment (leidwang) [reply qemu/tests/virtio_port_hotplug.py:95]: Okay,I also have doubts about this place. Last time I discussed this issue with @6-dehan. We suspected that it was the PID left over from the previous case, so We decided to kill them all.
@6-dehan @nanliu-r Could you explain some details about this case?

[the remark this answers is not in the document]
Comment (leidwang) [reply qemu/tests/virtio_port_hotplug.py:95]: After debugging, I found that the extra pid came from a skip case.

**Comment by leidwang** [comment]
@6-dehan @YongxueHong Could you help review this patch again?Thanks a lot.

**Comment by 6-dehan** [comment]
@leidwang @nanliu-r  @YongxueHong  LGTM
New code logic and test run results are fine from my side.

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Comment by nanliu-r** [comment]
> @leidwang @nanliu-r @YongxueHong LGTM
> New code logic and test run results are fine from my side.

ACK, thanks for debugging.

**Pull request merged**